Library/Formula/libcppa.rb

```diff
@@ -2,8 +2,11 @@ class Libcppa < Formula
   # TODO: since libcppa has been renamed to CAF, this formula should eventually
   # be renamed to 'caf.rb'.
   homepage "http://actor-framework.org/"
-  url "https://github.com/actor-framework/actor-framework/archive/0.13.tar.gz"
-  sha1 "80de2666cad9795f280994f6dd12aae8368e65fe"
+  # For this release (0.13.1) we do not use a github tag because we've added
+  # mountain-lion compatibility fix discovered by brew-bot.
```

Comment:
These are both fair points. Instead of releasing a patch, we shipped a hotfix release `0.13.2` and will file a new PR with that version.